avm/res/compute/virtual-machine/tests/e2e/windows.disks/dependencies.bicep

```diff
@@ -4,12 +4,37 @@ param virtualNetworkName string
 @description('Optional. The location to deploy to.')
 param location string = resourceGroup().location
 
-@description('Required. The name of the disk to create.')
+@description('Required. The name of the data disk to create.')
 param sharedDiskName string
 
+@description('Required. The name of the os disk to create.')
+param osDiskName string
+
+@description('Required. The name of the VM to take the OS disk from. It\'s not possible to create an OS disk without a VM.')
+param osDiskVMName string
+
+@description('Required. The name of the Disk Encryption Set to create.')
+param diskEncryptionSetName string
+
+@description('Required. The name of the Key Vault to create.')
+param keyVaultName string
+
+@description('Required. The name of the Managed Identity to create.')
+param managedIdentityName string
+
+@description('Optional. The password to leverage for the login.')
+@secure()
+param password string = newGuid()
+
+@description('Required. The name of the deployment script that waits for a role assignment to propagate.')
+param waitDeploymentScriptName string
+
+@description('Specifies the name of the administrator account. <br><br> This property cannot be updated after the VM is created. <br><br> **Windows-only restriction:** Cannot end in "." <br><br> **Disallowed values:** "administrator", "admin", "user", "user1", "test", "user2", "test1", "user3", "admin1", "1", "123", "a", "actuser", "adm", "admin2", "aspnet", "backup", "console", "david", "guest", "john", "owner", "root", "server", "sql", "support", "support_388945a0", "sys", "test2", "test3", "user4", "user5". <br><br> **Minimum-length (Linux):** 1  character <br><br> **Max-length (Linux):** 64 characters <br><br> **Max-length (Windows):** 20 characters.')
```

Comment:
Remove the "<br><br>" from the description, ignore the comment if this is directly required for the readme file